android/cli/commands/_build.js

```diff
@@ -1530,8 +1530,8 @@ AndroidBuilder.prototype.run = async function run(logger, config, cli, finished)
 		await this.checkIfNeedToRecompile();
 
 		// Notify plugins that we're prepping to compile.
-		await new Promise((resolve) => {
-			cli.emit('build.pre.compile', this, resolve);
+		await new Promise((resolve, reject) => {
+			cli.emit('build.pre.compile', this, e => (e === null ? resolve() : reject(e)));
```

Comment:
Not saying you have to change this, but I always do `e => e ? reject(e) : resolve()` just in case `e` not null, but still falsey.